src/camera/README.md

```diff
@@ -0,0 +1,19 @@
+# Camera
+
+The camera system in SM64 is notoriously complex, with the original file exceeding 11000 lines of code. HackerSM64 separates the camera code into more relevant and granular files. An index of changes is provided below.
```

Comment:
Not sure how I feel about a readme in the middle of source files lol

Reply:
tbf this is a big enough change to warrant one (at the very least to prevent some `where camera` issues from people who dont use the server)